model_training/local_prediction/yolo_object_detection.py

```diff
@@ -67,18 +85,22 @@
                 class_ids.append(class_id)
 
     indexes = cv2.dnn.NMSBoxes(boxes, confidences, 0.5, 0.4)
-    print(indexes)
     font = cv2.FONT_HERSHEY_PLAIN
-    for i in range(len(boxes)):
-        if i in indexes:
-            x, y, w, h = boxes[i]
-            label = str(classes[class_ids[i]])
-            color = colors[class_ids[i]]
-            cv2.rectangle(img, (x, y), (x + w, y + h), color, 2)
-            cv2.putText(img, label, (x, y + 30), font, 1, color, 2)
-
+    f = open(labelfile + ".txt", "w+")
```

Comment:
Can you add a `.gitignore` that ignores `.txt` files inside test_images folder? This is because I think this feature is good but not necessary to push the output files to repo